src/components/sidekick/index.test.tsx

```diff
@@ -33,13 +67,16 @@ describe('Sidekick', () => {
   });
 
   it('renders sidekick when panel tab is clicked', () => {
-    const updateLayout = jest.fn();
-    const wrapper = subject(updateLayout);
+    const updateSidekick = jest.fn();
+    const wrapper = subject({ updateSidekick });
 
     const ifAuthenticated = wrapper.find(IfAuthenticated).find({ showChildren: true });
```

Comment:
We should be able to simplify the wrapper lookups by removing `ifAuthenticated`.